### ISSUE TITLE
Depending on the package upath@1.0.4 incompatible with node v10

### ISSUE THREAD

**Issue by stomvi** [relayjs/relay-examples#77]
Using `node` `v10.1.0`

`yarn install` shows:
```
error upath@1.0.4: The engine "node" is incompatible with this module. Expected version ">=4 <=9".
```

Please update this dependency to version 1.1.0.

**Comment by stomvi** [comment]
For someone who installed `node` by `brew` on Mac, just unlink the `node` and install node@8:
```
brew unlink node
brew install node@8
```

Then since node@8 is an alternative version, add the path to your .bash_profile. For example:
```
echo 'export PATH="/usr/local/opt/node@8/bin:$PATH"' >> ~/.bash_profile
source ~/.bash_profile
```

**Comment by kassens** [comment]
Should be fixed by #90.
